libfreerdp/core/nego.c

```diff
@@ -36,34 +36,6 @@
 
 #define TAG FREERDP_TAG("core.nego")
 
-struct rdp_nego
```

Comment:
@kubistika I can't remember why we did this

Reply:
i did that so we can get the routing token from our code.

Reply:
So can we revert it

Reply:
No, we use rdpNego struct in pf_server to get the routing token and extract target host information. This struct should remain exported.